readthedocs/restapi/views/footer_views.py

```diff
@@ -40,10 +41,15 @@ def get_version_compare_data(project, base_version=None):
         'project': six.text_type(highest_version_obj),
         'version': six.text_type(highest_version_comparable),
         'is_highest': True,
+        'is_stable': False,
     }
     if highest_version_obj:
         ret_val['url'] = highest_version_obj.get_absolute_url()
         ret_val['slug'] = (highest_version_obj.slug,)
+    if base_version and base_version.slug == STABLE:
+        ret_val['is_highest'] = False
+        ret_val['is_stable'] = True
```

Comment:
I'm not really following the whole flow, but supposing this case:

* `base_version` is the STABLE
* we get a `highest_version_obj`

we end up returning a `is_stable` in `True` but with the `url` and `slug` from another version?

(I'm asking because I'm a little confused here)

Reply:
Well, I'm confused too p: but looking to the docstring, says:

> Retrieve metadata about the highest version available for this project.

So, always return that information about the highest. And when `base_version` is given, `is_stable` and `is_highest` are for that version. I think that method needs refactoring.

And I realized that when I enter to stable version manually (docs/2018 instead of docs/stable in my example) the message appears, I need to check that case.

Reply:
After looking deeper into the code, I found that `highest` == `stable`, or at least the function return the mayor version (except for latest, stable) of the project.

@humitos can you confirm that? In that case, we need to add some field `is_latest`, no `is_stable`, or keep the names of the current PR and change the logic (I'm not sure if this breaks other part of the code that uses the API).